packages/react-native-web/src/modules/useLocale/index.js

```diff
@@ -6,6 +6,7 @@
  *
  * @flow strict
  */
+'use client';
```

Comment:
Can this move to [`exports/useLocaleContext`](https://github.com/necolas/react-native-web/tree/master/packages/react-native-web/src/exports/useLocaleContext)?